Sources/KotlinStandAlone/Generator.swift

```diff
@@ -0,0 +1,111 @@
+//
+// github.com/screensailor 2022
+//
+
+import Lexicon
+import UniformTypeIdentifiers
+
+public enum Generator: CodeGenerator {
+	
+	// TODO: prefixes?
+	
+	public static let utType = UTType.sourceCode
+	
+	public static func generate(_ json: Lexicon.Graph.JSON) throws -> Data {
+		guard let o = json.swift().data(using: .utf8) else {
```

Comment:
These days it's better to use the non-failable initialiser for Data:
```suggestion
	let o = Data(json.kotlin().utf8)
```

Reply:
Done. tbh i just copy pastaed the Swift version so might be worth making the updates there as well